apparmor.d/groups/browsers/falkon

```diff
@@ -0,0 +1,215 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2024 Alexandre Pujol <alexandre@pujol.io>
+# Copyright (C) 2024 Besanon  <m231009ts@mailfence.com>
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{name} = falkon{,.sh,-wayland}
+@{exec_pathFFal} = @{bin}/falkon
+profile falkon @{exec_pathFFal} {
+  include <abstractions/base>
+  include <abstractions/audio-client>
+  include <abstractions/bus-session>
+  include <abstractions/bus-system>
+  include <abstractions/bus/org.a11y>
+  include <abstractions/bus/org.freedesktop.FileManager1>
+  include <abstractions/bus/org.freedesktop.login1>
+  include <abstractions/bus/org.freedesktop.portal.Desktop>
+  include <abstractions/bus/org.freedesktop.RealtimeKit1>
+  include <abstractions/bus/org.gtk.Private.RemoteVolumeMonitor>
+  include <abstractions/dconf-write>
+  include <abstractions/desktop>
+  include <abstractions/enchant>
+  include <abstractions/lxqt>
+  include <abstractions/fontconfig-cache-read>
+  include <abstractions/graphics-full>
+  include <abstractions/gstreamer>
+  include <abstractions/nameservice-strict>
+  include <abstractions/ssl_certs>
+  include <abstractions/qt5-shader-cache>
+  include <abstractions/thumbnails-cache-read>
+  include <abstractions/thumbnails-cache-write>
+  include <abstractions/user-download-strict>
+  include <abstractions/user-read-strict>
+
+  network inet dgram,		# essential
+  network inet stream,		# essential
+
+  network inet6 stream,   	# Not needed
+  network inet6 dgram,    	# Not needed
+  network inet raw,      	# Not needed
+  network inet6 raw,		# Not needed
+  network netlink raw,		# Not needed
+  network packet dgram,		# Not needed
```

Comment:
If there are not needed, then do not put them...

The following int6 are needed for IPV6: 
```
  network inet6 dgram,
  network inet6 stream,
```